### PR TITLE
Add CI

### PR DESCRIPTION
**Pull request by phillipwright7**
## Summary

Adding continuous integration to my calculator project.

## Description

This PR is designed to add CI to my calculator project in order to continuously execute and test with my `calculator_test.go` file.

### CODE REVIEW

**Review by jboursiquot — CHANGES_REQUESTED** [review]
You have a conflict.

**Comment by phillipwright7** [comment]
Conflict fixed.

File: Makefile [hunk 1]
```diff
@@ -20,4 +20,13 @@ run-marvel:
 	cd cmd/marvel && go run main.go -p=../../.env
 
 run-seeder:
+	cd cmd/seeder && go run main.go -db=../../products.db -json=../server/products.json
+
+calculator-build:
+	cd cmd/calculator
+	go build -v ./...
```
Comment (jboursiquot) [inline Makefile:27]: Why the `-v ./...` here. Not needed. Also, you'll want to combine those two commands on one line.

File: Makefile [hunk 2]
```diff
@@ -20,4 +20,13 @@ run-marvel:
...
+
+calculator-test:
+	cd calculator
```
Comment (jboursiquot) [inline Makefile:30]: Combine those two commands on one line. Also, you already have a `run-seeder` Make target. No need to repeat that here again.

**Review by jboursiquot — CHANGES_REQUESTED** [review]

**Review by jboursiquot — APPROVED** [review]

**Pull request merged**